### PR TITLE
Package ez_api.2.0

### PR DESCRIPTION
**Pull request by hernoufM**
### `ez_api.2.0`
Easy API library and tools

---
* Homepage: https://github.com/ocamlpro/ez_api
* Source repo: git+https://github.com/ocamlpro/ez_api.git
* Bug tracker: https://github.com/ocamlpro/ez_api/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/ez_api/ez_api.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+synopsis: "Easy API library and tools"
+maintainer: "Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"
+authors: [
+  "Cagdas Bozman <cagdas.bozman@ocamlpro.com>"
+  "Michael Laporte <michael.laporte@ocamlpro.com>"
+  "Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"
+  "Maxime Levillain <maxime.levillain@ocamlpro.com>"
+  "Alain Mebsout <alain.mebsout@ocamlpro.com>"
+]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
```
Comment (kit-ty-kate) [inline packages/ez_api/ez_api.2.0/opam:11]:
```suggestion
license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
```

**Comment by hernoufM** [comment]
Thank you!! I was just about to update the PR and it was done for me =)

**Comment by kit-ty-kate** [comment]
I was about to merge it but i noticed something weird:
* the package has version `2.0`
* but the tag says: version `2.0.0`

Is this expected?

**Comment by hernoufM** [comment]
Ah no, actually it's not something expected, I'll fix it up

**Pull request closed without merging**